ediarum.earlyprint.edit.framework

```diff
@@ -6691,6 +6691,68 @@ parent::not(p)</String>
 											<null/>
 										</field>
 									</poPatch>
+									<poPatch>
+										<field name="fieldPath">
+											<String>authorExtensionDescriptor/actionDescriptors</String>
+										</field>
+										<field name="index">
+											<Integer>273</Integer>
+										</field>
+										<field name="value">
+											<action>
+												<field name="id">
+													<String>hi[color]</String>
+												</field>
+												<field name="name">
+													<String>Hervorhebungen(Usereingabe)</String>
+												</field>
+												<field name="description">
+													<String>In gedrucktem Text markieren</String>
+												</field>
+												<field name="largeIconPath">
+													<String></String>
+												</field>
+												<field name="smallIconPath">
+													<String></String>
+												</field>
+												<field name="accessKey">
+													<String></String>
+												</field>
+												<field name="accelerator">
+													<null/>
+												</field>
+												<field name="actionModes">
+													<actionMode-array>
+														<actionMode>
+															<field name="xpathCondition">
+																<String>ancestor-or-self::text</String>
+															</field>
+															<field name="argValues">
+																<serializableOrderedMap>
+																	<entry>
+																		<String>fragment</String>
+																		<String>&lt;hi xmlns="http://www.tei-c.org/ns/1.0" rendition="${ask('Color', String)}">&lt;/hi></String>
```

Comment:
Mach lieber style statt rendition, in css Syntax, also: style="color: {ask('Color', String)}"

Reply:
Dann auch in dem Namen auf Schriftfarbe reduzieren